src/dmd/tokens.d

```diff
@@ -709,6 +709,89 @@ extern (C++) struct Token
 
 nothrow:
 
+    /**
+     * Tests the receiver and the given value for equality.
```

Comment:
What's the `receiver` and what does it have to do with opEquals?

Reply:
In the following snippet, `foo` is the receiver: `foo.opEquals(bar)`.